comparison.md

```diff
@@ -8,19 +8,19 @@ When you're starting out with modding, it's important to use a template. Templat
+The mirin template is high-level compared to exschwasion template. I don't mean high-level as in more complex, or better than exsch. I'm talking in terms of abstraction. With the exschwasion template, you'll learn to become familiar with things like activation rates, apply modifiers, and the behavior of `clearall`. You'll be able to read the template code to figure out how the code you write gets transformed into the mods that are actually applied to the game. You'll be working with code closer to the interface that notitg provides to apply mods. However, in the mirin template, there's a non-trivial layer of abstraction on top of notitg's interface. On one hand, this makes it easier to express complex concepts using the mirin template without worrying about lower-level concepts like sustain times and activation rates and mod strings. But on the other hand, because it introduces a layer of abstraction, it's not as easy to get a strong mental model of what's actually going on in the template. Also, the creator of the mirin template (me)(Xero) has learned enough computer science to be considered dangerous, and the source code of the mirin template is definitely dangerous. If you were to edit the mirin template without any prior knowledge of lua or sm5, there are a bunch of concepts that you won't have learned just by using the mirin template, and it's relatively opaque. Meanwhile, the exsch template and others are usually much more straightforward in implementation, which makes it easy to learn more. When you're using the mirin template though, I don't recommend actually looking at the source. It's much easier to ignore, because it's in a whole separate folder, instead of being in the template with a whole "don't touch this kiddo" block, which makes it more confusing.
```

Comment:
I don't think the addition of "knowledge of lua or sm5" is correct here. The point of the sentence isn't to explain that prior knowledge is necessary. I'm trying to explain that there is a knowledge gap between interface and the implementation that isn't usually present in other templates. Other templates have interfaces that follow very simply from their implementation, but in mirin I have done a much more interface-first design that creates a gap here.
I don't think sm5 knowledge is relevant at all  (perhaps you meant NotITG), and knowledge of notitg/lua isn't the type of information I'm trying to describe. Maybe that sentence is bad and needs to be more clear, but this is not the right way to clarify.

The other changes here are good though.

Reply:
oh wait I meant sm3.95 but yeag notitg too

Reply:
I don't want to mean either of them in that particular sentence

Reply:
these sorts of diffs that are hard to look at are the reason we want the markdown files to be wrapped